api/reporting/schemas/input/descriptions/user.yaml

```diff
@@ -59,3 +59,45 @@ user:
   username: |
     User's GSuite username, including appropriate domain for this environment. Uniquely describes a
     user account (but not constrained to be unique by BigQuery).
+  institution_id: |
```

Comment:
Note: this is_only_ used by the generator tool, so eventually this file can be eliminated.

Reply:
Right, and I also feel this is not too helpful. I end with edit swagger yaml first then copy back. Just to keep it consitent.